salt/auth/ldap.py

```diff
@@ -362,10 +362,10 @@ def groups(username, **kwargs):
             search_results = bind.search_s(search_base,
                                            ldap.SCOPE_SUBTREE,
                                            search_string,
-                                           [_config('accountattributename'), 'cn'])
+                                           [_config('accountattributename'), 'cn', _config('groupattribute'), 'cn'])
 
```

Comment:
log.critical('SEARCH_RESULTS: %s', search_results)

Reply:
@DmitryKuzmenko I'll set up a test server to talk to our freeipa server and get back to you soon,